modules/claims_api/app/sidekiq/claims_api/report_hourly_unsuccessful_submissions.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class ReportHourlyUnsuccessfulSubmissions < ClaimsApi::ServiceBase
+    sidekiq_options retry: 7
+
+    def perform
+      return if skip_processing?
+
+      @search_to = Time.zone.now
+      @search_from = 1.hour.ago
+      @reporting_to = Time.now.in_time_zone('Eastern Time (US & Canada)').strftime('%l:%M%p %Z')
+      @reporting_from = 1.hour.ago.in_time_zone('Eastern Time (US & Canada)').strftime('%l:%M%p %Z')
+      @errored_claims = ClaimsApi::AutoEstablishedClaim.where(created_at: @from..@to, status: 'errored').pluck(:id).uniq
+      @errored_poa = ClaimsApi::PowerOfAttorney.where(created_at: @from..@to, status: 'errored').pluck(:id).uniq
+      @errored_itf = ClaimsApi::IntentToFile.where(created_at: @from..@to, status: 'errored').pluck(:id).uniq
+      @errored_ews = ClaimsApi::EvidenceWaiverSubmission.where(created_at: @from..@to,
+                                                               status: 'errored').pluck(:id).uniq
+
+      if errored_submissions_exist?
+        notify(
+          @errored_claims,
+          @errored_poa,
+          @errored_itf,
+          @errored_ews,
+          @reporting_to,
+          @reporting_from
+        )
+      end
+    end
+
+    # rubocop:disable Metrics/ParameterLists
+    def notify(claims, poa, itf, ews, from, to)
+      ClaimsApi::Slack::FailedSubmissionsMessenger.new(
+        claims,
+        poa,
+        itf,
+        ews,
+        to,
+        from
+      ).notify!
+    end
+    # rubocop:enable Metrics/ParameterLists
+
+    private
+
+    def errored_submissions_exist?
+      [@errored_claims, @errored_poa, @errored_itf, @errored_ews].any? { |var| var.count.positive? }
+    end
+
+    def skip_processing?
+      !Rails.env.production?
+    end
```

Comment:
I think your goal here is to only set this in the "production" API but this stands true in all environments. We'll need to trigger this off of a settings value. 
For example, staging
```
irb(main):001:0> Rails.env
=> "production"
```

ex: https://github.com/department-of-veterans-affairs/vets-api/blob/e66ef88d7c87d445ce5471bbec310cac005967dc/modules/claims_api/app/sidekiq/claims_api/claim_auditor.rb#L6

Reply:
sorry, the intent here was to use the existing value as it's only set in production. We could create a new settings value like you have in the new changes, but I'd probably just roll with the existing value unless you have a specific reason to create a new one. 

https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/blob/2621bf3b0934b07b7e33947c4053147ede660004/apps/vets-api/prod/values.yaml#L2294

Reply:
@FonzMP This makes sense, updated to utilize that value.